libkbfs/block_retrieval_worker.go

```diff
@@ -57,7 +57,12 @@ func (brw *blockRetrievalWorker) HandleRequest() (err error) {
 	}
 
 	// Create a new block of the same type as the first request
-	block := retrieval.requests[0].block.NewEmpty()
+	var block Block
+	retrieval.reqMtx.RLock()
+	if retrieval.requests[0].block != nil {
```

Comment:
Hmm. Apparently we have at least one test checking for this. WDYT @strib?
https://ci.keyba.se/job/kbfs/job/PR-480/22/execution/node/238/log/